README.md

```diff
@@ -1,2 +1,21 @@
 # hello-world
-Intro tutorial repository
+
+<h1>Intro tutorial repository</h1> <p> this somehow wrote twice? somehow "# hello-world" in bold makes the initial title thingie with a heading and caption, then I can write whatever I want. Only difference = the auto title has a red left border and what I write has a green left border. </p> 
+
+
+
+<ul style="list-style-type: square"> Other styles: 
+<li>"hello-world" heading =/= any left border</li> 
+<li><ul>headings all have light grey bottom border that doesn't meet up with the one on top: thinking of how to do that... 
+    <li>possibility: can you have multiple borders in CSS3?</li>
+    <li>Can you stop a border before it meets the edge?</li>
+    <li>Can you have different borders around the content vs. padding (maybe using "content-box" vs "border-box" type thing)</li>
+    </ul>
+    </li>
```

Comment:
Can add comments under specific lines, and comments can be individual or a "review"